src/Illuminate/Http/Client/Response.php

```diff
@@ -27,6 +27,20 @@ class Response implements ArrayAccess
      */
     protected $decoded;
 
+    /**
+     * The request cookies.
+     *
+     * @var \GuzzleHttp\Cookie\CookieJar
```

Comment:
It's odd since on `PendingRequest` this is an `array`. Not sure why it's different here.